docs/javascript/browser-apis-and-jquery.md

```diff
@@ -1345,3 +1345,94 @@ jQuery's `text()` method allows you to get or set the text of an HTML element.
 ![text method example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/textMethod.gif?raw=true){ width="100%" }
 <figcaption></figcaption>
 </figure>
+
+## Styling DOM Nodes
+
+### `element.style` 
+
+The `style` property allows you to set a style to a given element. 
+
+Note: Javascript syntax for accessing style properties differs from css syntax:
```

Comment:
Make sure to use the correct naming conventions for languages.

- JavaScript
- CSS
- HTML

I think we could be explicit here and tell them *how* JavaScript's syntax differs from standard CSS syntax through a few examples. Anything that is dashed-case in CSS turns to camelCase in JS. Maybe a table would work? You can look at how I've set those up on the comparison operators section in the JS language docs.

Also, for notes, there's a feature I can turn on called [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#configuration) that we may be able to use here. I'll make a note to self to try this after we merge your changes.

Reply:
Was able to add the "Admonitions" feature. So, you can do this here:

```text
!!! note

    JavaScript syntax for accessing style properties differs from CSS syntax.
```